examples/algorithms/retry.hpp

```diff
@@ -101,7 +101,7 @@ struct _retry_sender {
   }
 
   template <class Env>
-  friend auto tag_invoke(stdex::get_sender_traits_t, const _retry_sender&, Env)
```

Comment:
The `get_sender_traits_t` type needs to be available. IIRC, LEWG asked for the tags to have name-able types.

Reply:
All the tag types?
- `set_value_t`, `set_error_t`, `set_stopped_t`
- `connect_t`, `start_t`, `schedule_t`, ...
- `transfer`, `let_value`, `bulk`, ...

It means, I also need to change specification to show this.

Reply:
Yes